.github/workflows/go.yml

```diff
@@ -13,7 +13,23 @@ env:
   go-cache-name: go
 
 jobs:
+  check-changed-files:
```

Comment:
I would recommend naming that job `check-changes` for consistency with other Github workflows